docs/common-problems.asciidoc

```diff
@@ -55,3 +55,67 @@ you can try to:
 * <<adjust-concurrent-requests>>
 * <<add-apm-server-nodes>>
 
+[float]
+[[ssl-client-fails]]
+=== SSL client fails to connect to Logstash?
```

Comment:
Is all of this really logstash specific? Can't you also run into SSL issues with ES, Kibana, Kafka?

Reply:
Agreed, this was copied over from beats but now that you point it out it is odd.  removed